tests/end2end/features/misc.feature

```diff
@@ -671,3 +671,38 @@ Feature: Various utility commands.
         And I wait until the scroll position changed
         And I run :click-element id link
         Then the error "Element position is out of view!" should be shown
+
+    ## :command-history-{prev,next}
+    
+    Scenario: Calling previous command
+        When I run :set-cmd-text :message-info blah
+        And I run :command-accept
+        And I wait for "blah" in the log
+        And I run :set-cmd-text :
+        And I run :command-history-prev
+        And I run :command-accept
+        Then the message "blah" should be shown
+    
+    Scenario: Browsing through commands 
+        When I run :set-cmd-text :message-info blarg
+        And I run :command-accept
+        And I wait for "blarg" in the log
+        And I run :set-cmd-text :
+        And I run :command-history-prev
+        And I run :command-history-prev
```

Comment:
To be precise, they don't fail, they end with error:
`ERROR tests/end2end/features/test_misc_bdd.py::test_calling_previous_command_when_history_is_empty`
`ERROR tests/end2end/features/test_misc_bdd.py::test_calling_next_command_when_theres_no_next_command`

And yes, I can reproduce them.

But apart from that you were right, browsing through commands [fails](https://travis-ci.org/The-Compiler/qutebrowser/jobs/165624171).